src/components/Positions/Positions.columns.js

```diff
@@ -212,7 +212,7 @@ export default function getColumns(props) {
       copyText: rowIndex => filteredData[rowIndex].marginFunding,
     },
     {
-      id: 'swapType',
+      id: 'marginFundingType',
```

Comment:
Why this changed?

Reply:
reused already existing list of columns https://github.com/bitfinexcom/bfx-report-ui/pull/322/files#diff-7b76267d52037735bda922404034396aR56, but in some cases the id's didn't correspond to data attributes